### PR TITLE
feature: Correct register form

### PR DESCRIPTION
**Pull request by JimTheCat**
- Adapt register form with new values
- Correct translations
- Adapt static gender values

Refs: CU-8697369r5

### CODE REVIEW

**Comment by JimTheCat** [comment]
Task linked: [CU-8697369r5 Correct register form](https://app.clickup.com/t/8697369r5)

File: frontend/src/Pages/Register/Register.tsx [hunk 1]
```diff
@@ -26,63 +23,66 @@ import 'dayjs/locale/en';
...
-    { re: /[$&+,:;=?@#|'<>.^*()%!-]/, label: t('card.password.requirement.special') },
+    {re: /[0-9]/, label: t('card.password.requirement.number')},
+    {re: /[a-z]/, label: t('card.password.requirement.lowercase')},
+    {re: /[A-Z]/, label: t('card.password.requirement.uppercase')},
+    {re: /[$&+,:;=?@#|'<>.^*()%!-]/, label: t('card.password.requirement.special')},
   ];
 
   const checks = requirements.map((requirement, index) => (
-    <PasswordRequirement key={index} label={requirement.label} meets={requirement.re.test(passwordValue)} />
+    <PasswordRequirement key={index + 1} label={requirement.label} meets={requirement.re.test(passwordValue)}/>
```
Comment (KinTrae) [inline frontend/src/Pages/Register/Register.tsx:62]: Why index + 1?

File: frontend/src/Pages/Register/Register.tsx [hunk 2]
```diff
@@ -107,148 +107,170 @@ export const Register = () => {
...
+            </Grid.Col>
+            <Grid.Col span={1}>
+              <DatePickerInput
+                withAsterisk
+                required
+                value={date}
+                onChange={
+                  (value) => {
+                    setDate(value);
+                    form.setFieldValue('age', value);
```
Comment (KinTrae) [inline frontend/src/Pages/Register/Register.tsx:180]: and here

File: frontend/src/Translations/en/register.json [hunk 3]
```diff
@@ -30,9 +30,10 @@
 				"uppercase": "Includes uppercase letter"
 			}
 		},
-		"repeatemail": {
-			"label": "Repeat email",
-			"placeholder": "Repeat your email"
+		"uniqueLogin": {
+			"invalid": "Tag can not contain white chars",
```
Comment (KinTrae) [inline frontend/src/Translations/en/register.json:34]: i know that 'can not' and 'cannot' is the same, but 'cannot' is more common so please use this

File: frontend/src/Translations/pl/login.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
 {
 	"card": {
 		"email": {
-			"invalid": "Niepoprawny email",
-			"label": "Email",
-			"placeholder": "Wprowadź swój email"
+			"invalid": "Niepoprawny tag bądź email",
+			"label": "Tag / Email",
```
Comment (KinTrae) [inline frontend/src/Translations/pl/login.json:5]: please be consistent. Either use 'lub' or 'bądź'

**Review by KinTrae — APPROVED** [review]
Great job😎

**Pull request merged**